src/Imgix/UrlHelper.php

```diff
@@ -29,8 +36,8 @@ public function setParameter($key, $value) {
         }
     }
 
-    public function deleteParamter($key) {
-        $this->deleteParamter($key, "");
+    public function deleteParameter($key) {
```

Comment:
Ooh, a bonus fix. Thanks for catching this.

Reply:
You're welcome!
Requested changes done.